devel/ansible/roles/dev/files/motd

```diff
@@ -8,6 +8,7 @@ bstart:     Start the Bodhi service.
 bstop:      Stop the Bodhi service.
 btest:      Run Bodhi's test suite.
 bteststyle: Run just the flake8 code style tests.
+bfedmsg:    View all fedmsgs Bodhi has emitted.
```

Comment:
I think it's best to delete the double docs, so I filed a PR for that:

https://github.com/fedora-infra/bodhi/pull/1527